r2i/onnxrt/engine.cc

```diff
@@ -28,7 +28,7 @@ namespace onnxrt {
 Engine::Engine () {
   /* Initialize all variable members */
   logging_level = OrtLoggingLevel::ORT_LOGGING_LEVEL_WARNING;
-  intra_num_threads = 1;
+  intra_num_threads = 0;
```

Comment:
Just to understand, what is the reason for this change?

Reply:
From my tests I constantly encounter that a better performance is obtained when this property is set to 0, I believe this optimization should be by default.